internal/api/auth.go

```diff
@@ -349,7 +349,7 @@ func (w *OidcCredential) getAssertion(ctx context.Context) (string, error) {
 		return "", fmt.Errorf("getAssertion: cannot unmarshal response: %v", err)
 	}
 
-	if tokenRes.Value == nil {
+	if tokenRes.Value == nil || tokenRes.Count == nil {
```

Comment:
AI found a good problem, but suggested a bad fix.  problem is with `tokenRes`
```suggestion
	if tokenRes == nil || tokenRes.Value == nil {
```